src/CommunityToolkit.Maui.Core/Extensions/UIViewExtensions.macios.cs

```diff
@@ -84,4 +84,30 @@ public static class UIViewExtensions
 		UIDevice.CurrentDevice.CheckSystemVersion(11, 0)
 			? view.SafeAreaLayoutGuide.WidthAnchor
 			: view.WidthAnchor;
+
+	/// <summary>
+	/// Get Key Window
+	/// </summary>
+	public static UIWindow? GetKeyWindow()
```

Comment:
Why not use `Microsoft.Maui.Platform.UIApplicationExtensions.GetKeyWindow` method?

Reply:
it is internal, I didn't know about it.